misc/qutebrowser.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 suve <veg@svgames.pl> -->
+<component type="desktop">
+	<id>qutebrowser.desktop</id>
```

Comment:
Looking at [the docs](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic), this should probably be `org.qutebrowser.qutebrowser`.

Reply:
I've set this to `org.qutebrowser.qutebrowser.desktop`, since omitting the `.desktop` part at the end causes appstream-util to complain.
```
$ appstream-util validate-relax misc/qutebrowser.appdata.xml 
misc/qutebrowser.appdata.xml: FAILED:
• markup-invalid: <id> does not have correct extension for kind
```

Reply:
Looking at https://github.com/hughsie/appstream-glib/issues/146 that was with an earlier version of that standard, before `<launchable>` existed. Since the recommendation to have `<id>` match the desktop filename seems to be gone now, I guess it'd still be better to remove it.

Reply:
Hm. I guess then maybe Fedora hasn't updated `appstream-util` in its repos. I can remove the suffix.